sdk/metric/view/view_test.go

```diff
@@ -371,8 +372,12 @@ func TestViewTransformAttributes(t *testing.T) {
 				WithFilterAttributes(tt.filter...),
 			)
 			require.NoError(t, err)
-
-			got := v.TransformAttributes(inputSet)
+			filter := v.AttributeFilter()
+			if tt.wantNilFilter {
+				assert.Nil(t, filter)
+				return
+			}
+			got := filter(inputSet)
```

Comment:
The panic should be guarded against here.

```suggestion
			} else {
				require.NotNil(t, filter)
			}
			got := filter(inputSet)
```

Reply:
Did this, and move the nil login into its own test. Also fixed test name.